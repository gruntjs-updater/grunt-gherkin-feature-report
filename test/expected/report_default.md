## Medical Record Numbers

- Manage a patient's medical record numbers
- Show a relevant patient MRN in the team header


## Team Patient Summary
> This is the feature description.
> Second line of description.

- Update the patient summary as a provider
- Request to update the patient summary as a patient
- View a patient summary updated by a different team member


