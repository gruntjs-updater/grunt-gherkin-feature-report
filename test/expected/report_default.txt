## Medical Record Numbers
- Manage a patient's medical record numbers
- Show a relevant patient MRN in the team header


## Team Patient Summary
- Update the patient summary as a provider
- Request to update the patient summary as a patient
- View a patient summary updated by a different team member


